src/app/components/appendTask.js

```diff
@@ -0,0 +1,13 @@
+export default function (index, todoList) {
+  const li = document.createElement('li');
+  const task = todoList.getTask(index);
+  li.innerHTML = `<div>
+  <i class="fas fa-check"></i>
```

Comment:
- [ ] It would be great if this part is an interactive checkbox. This because in this way we can check a task complete or have it empty if we don't have finished it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/125553871-d2fc7177-7afc-4b26-bb34-54c24992efb9.png)